### PR TITLE
[WIP] first toy for communicating to EOS through grpc

### PR DESCRIPTION
**Pull request by ffurano**
This PR will contribute with a GRPC connector to EOS instead of the current connector that forks a new shell process and parses stdout/stderr.

### CODE REVIEW

File: pkg/eosclientgrpc/eos_grpc/eos_grpc.pb.go [hunk 1]
```diff
@@ -0,0 +1,3182 @@
...
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// In applying this license, CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+// Code generated by protoc-gen-go. DO NOT EDIT.
+// source: eos_grpc.proto
```
Comment (butonic) [inline pkg/eosclientgrpc/eos_grpc/eos_grpc.pb.go:20]: could you add a link to the source?

Comment (butonic) [reply pkg/eosclientgrpc/eos_grpc/eos_grpc.pb.go:20]: is it this one: https://gitlab.cern.ch/eos/grpc-proto/-/blob/master/protobuf/Rpc.proto ?

Comment (ffurano) [reply pkg/eosclientgrpc/eos_grpc/eos_grpc.pb.go:20]: Hi, the source is in the same directory by now, with a comment showing the command line used to compile it.

**Comment by ffurano** [comment]
:-)

**Pull request merged**